resources/assets/js/src/components/chart/Chart.tsx

```diff
@@ -155,7 +164,7 @@ export default class Chart extends React.Component<IProps, IState> {
     private loadDataToChart(direction: string = null): void {
         let diff = 0;
 
-        if (this.state.dateRange.dateFrom !== null && this.state.dateRange.dateTo !== null) {
+        if (this.state.dateRange.dateFrom !== null && this.state.dateRange.dateTo !== null && !this.state.error) {
```

Comment:
Condition is getting to much complex

Reply:
what do you suggest? i need to check all this conditions

Reply:
you can separate date check to another function

Reply:
or to DateUtils